README.md

```diff
@@ -47,7 +47,8 @@ By default, CreateTestSbot deletes an existing database of the same `name` befor
 
 Valid `opts` keys include:
 - `name` *String* (optional) (default: `ssb-test + Number(new Date)`)
-    - `myTestName`: Sets the database in /tmp/myTestName 
+- `path` *String* (optional) (default: `/tmp/${name}`, where `name` is the above)
+    - `~/.ssb-test`: Sets the database in `~/.ssb-test`
```

Comment:
This sets the working location for the identity - log databases, secret, conn.json etc

Wonder if there's an easy word for that.
Anyway not a big deal